figmaTokens.json

```diff
@@ -0,0 +1,645 @@
+{
+  "letterSpacing": {
+    "very-tight": {
+      "value": "-1.5%",
+      "type": "letterSpacing"
+    },
+    "tight": {
+      "value": "-1%",
+      "type": "letterSpacing"
+    },
+    "semi-tight3": {
+      "value": "-0.5%",
+      "type": "letterSpacing"
+    },
+    "semi-tight2": {
+      "value": "-0.25%",
+      "type": "letterSpacing"
+    },
+    "semi-tight1": {
+      "value": "-0.15%",
+      "type": "letterSpacing"
+    },
+    "normal": {
+      "value": "0%",
+      "type": "letterSpacing"
+    },
+    "semi-wide": {
+      "value": "0.15%",
+      "type": "letterSpacing"
+    },
+    "wide": {
+      "value": "0.5%",
+      "type": "letterSpacing"
+    },
+    "very-wide": {
+      "value": "1%",
+      "type": "letterSpacing"
+    }
+  },
```

Comment:
@Ali-Khamis 
I got a feed back from Mostafa regarding `letter-spacing `, and the value will be relative to the font-size property 
so we had to do some conversion 

Ex : The value  `-.15%` => By dividing it by  `100` and add our unit `em` .
The final value will be  `-.0015em`

Reply:
@elhaw 
That way the numbers will be too low to do any noticeable changes.
My suggestion would be that we divide by 10 which gives noticeable changes.

Normal:
![Screenshot from 2022-06-16 15-07-53](https://user-images.githubusercontent.com/68826445/174076869-aa5b67c1-b977-4385-b6b9-938a7767b26d.png)

divide by 100:  very-wide = 0.01em
![Screenshot from 2022-06-16 15-07-44](https://user-images.githubusercontent.com/68826445/174076703-83c59371-08fd-4332-8b16-61b9ef9178ae.png)

divide by 10:  very-wide = 0.1em
![Screenshot from 2022-06-16 15-06-27](https://user-images.githubusercontent.com/68826445/174076787-7c732e30-3a31-440d-8109-dc1dd84f4fc0.png)

Reply:
You are right. No  noticeable difference between the normal letter spacing `(0em)` and `. 01em`  but it will be noticeable in case the font size is large such as `74px`.
We can add the values as per the design but we can discuss it with Mostafa later.